mlrun/feature_store/feature_set.py

```diff
@@ -223,6 +224,18 @@ def update_last_written_for_target(
                 target.last_written = last_written
 
 
+def emit_policy_to_dict(policy: EmitPolicy):
+    # Storey expects the policy to be converted to a dictionary with specific params and won't allow extra params
```

Comment:
Actually you can pass it the EmitPolicy directly https://github.com/mlrun/storey/blob/v1.0.1/storey/aggregations.py#L38

Reply:
Yes, it does, but when you go through `add_aggregations` the params have to be serialized, since this execution graph may be running remotely. The `EmitPolicy` class does not know how to serialize itself the way MLRun expects (i.e. implement `to_dict`), hence I'm always placing it in the `class_args` as a dictionary.